sdk/python/feast/client.py

```diff
@@ -1288,13 +1288,18 @@ def start_stream_to_online_ingestion(
             )
 
     def list_jobs(
```

Comment:
Should we have docs for this method @terryyylim?

Reply:
Yeah we should, added!